### ISSUE TITLE
Provide assets for PPSSPP

### ISSUE THREAD

**Issue by kaasknak** [lutris/lutris#3977]
### Bug description

When using lutris to play a PSP game the game crashes and gives corresponding errors in the terminal.

This has happened both with the unpatched and the patched game.
Having looked into the PPSSPP github for similar issues the closest issue I could find was this one https://github.com/hrydgard/ppsspp/issues/11343

Upon this I decided to compile PPSSPP myself and see if it would work.
Using PPSSPP I could start the game both patched and unpatched without any issues.

### How to Reproduce

1. Install using https://lutris.net/games/steins-gate/
2. Play game
3. Change to unpatched game to test
4. Try to play game again
5. 

### Expected behavior

Game starts.

### Log output

```shell
2022-01-04 12:56:35,513: Platform for Steins;Gate set to PSP
2022-01-04 12:56:39,404: Firmware 'PPSSPP/ppge_atlas.zim' not found!
Started initial process 93328 from gamemoderun /home/benji/.local/share/lutris/>
Start monitoring process.
/home/benji/.local/share/lutris/runners/retroarch/retroarch: /lib/x86_64-linux->
Monitored process exited.
Initial process has exited (return code: 256)
All processes have quit
Exit with return code 256
```

### System Information

```shell
[System]
OS:              Ubuntu 20.04 focal
Arch:            x86_64
Kernel:          5.11.0-44-generic
Desktop:         MATE
Display Server:  x11

[CPU]
Vendor:          GenuineIntel
Model:           Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz
Physical cores:  4
Logical cores:   8

[Memory]
RAM:             15.5 GB
Swap:            2.0 GB

[Graphics]
Vendor:          NVIDIA Corporation
OpenGL Renderer: NVIDIA GeForce GTX 1060/PCIe/SSE2
OpenGL Version:  4.6.0 NVIDIA 470.86
OpenGL Core:     4.6.0 NVIDIA 470.86
OpenGL ES:       OpenGL ES 3.2 NVIDIA 470.86
Vulkan:          Supported
```

### Checklist:

- [X] I'm not asking for support with a game or the wine runner.
- [X] I have followed the above mentioned guides and have all the graphics and wine dependencies installed.
- [X] I have checked for existing issues that describe my problem prior to opening this one.
- [X] I understand that improperly formatted bug reports may be closed without explanation.

**Comment by telanus** [comment]
A quick problem I see: "Firmware 'PPSSPP/ppge_atlas.zim' not found!" Also you tried with Libretro first. Did you follow https://docs.libretro.com/library/ppsspp/? Unlike the stand-alone PPSSPP emu, Libretro don't includethe bios files. That's why it won't work with Libretro but work with Stand-alone PPSSPP.

**Comment by kaasknak** [comment]
I am not sure.
The page you sent me seems to be about setting this up outside of lutris using a different wrapper for PPSSPP than Lutris or am I misunderstanding something?

**Comment by telanus** [comment]
I linked the libretro page as it states in your log it was using the libretro runner : Start monitoring process. /home/benji/.local/share/lutris/runners/retroarch/retroarch

**Comment by strycore** [comment]
This can be merged with issue #431 since it's about providing a way to load BIOS files in Retroarch
